frontend/src/components/contactsForm.tsx

```diff
@@ -0,0 +1,310 @@
+                name="last_name"
+                type="text"
+                autoComplete="on"
+                required
+                value={lastName}
+                onChange={handleLastNameChange}
+              ></input>
+
+              <label htmlFor="email">Email</label>
+              <input
+                className="email"
+                name="email"
+                type="text"
+                autoComplete="on"
+                required
+                value={email}
+                onChange={handleEmailChange}
+              ></input>
+
+              <label htmlFor="phone_number">Phone number</label>
+              <input
+                className="phone_number"
+                name="phone_number"
+                type="text"
+                autoComplete="on"
+                required
+                value={phoneNumber}
+                onChange={handlePhoneNumberChange}
+              ></input>
+
+              <label htmlFor="notes">Extra Notes</label>
+              <input
+                className="notes"
+                name="notes"
+                type="text"
+                autoComplete="on"
+                required
+                value={notes}
+                onChange={handleNotesChange}
+              ></input>
+
+              <label>
+                Relationship with Person
+                <input
+                  type="text"
+                  value={relationshipType}
+                  onChange={(event) => {
+                    const value = event.target.value
+                    if (/^[A-Za-z\s]*$/.test(value)) {
+                      setRelationshipType(value)
+                    }
+                  }}
+                />
+              </label>
+              <button type="button" onClick={addRelationship}>Add</button>
+              {relationships.map((type, index) => (
+                <div key={index}>
+                  <input
+                    type="text"
+                    value={type.relationshipType}
+                    onChange={(event) => { handleRelationshipChange(index, 'relationshipType', event.target.value) }
+                    }
+                  />
+                  <button type= "button" onClick={() => { handleRemoveRelationshipType(index) }}>
+                    x
+                  </button>
+                </div>
+              ))}
+
+              <label>
+                Date:
+                <input
+                  type="text"
+                  value={eventDate}
+                  onChange={(event) => {
+                    const value = event.target.value
+                    setEventDate(value)
+                  }}
+                />
+              </label>
+
+              <label>
+                Event:
+                <input
+                  type="text"
+                  value={eventType}
+                  onChange={(event) => {
+                    const value = event.target.value
+                    if (/^[A-Za-z\s]*$/.test(value)) {
```

Comment:
let's pull this regex into a top level global variable

Reply:
done!